minecraft-server/start-minecraft.sh

```diff
@@ -62,8 +62,8 @@ case "$TYPE" in
       ;;
     esac
 
-    rm -f $SERVER
-    wget -q https://getspigot.org$URL
+    #attempt https, and if it fails, fallback to http and download that way. Display error if neither works.
+    wget -q -N spigot_server.jar https://getspigot.org$URL || (echo "Falling back to http, unable to contact server using https..." && wget -q -N spigot_server.jar http://getspigot.org$URL) || echo "Unable to download new copy of spigot server"
```

Comment:
To keep the code better to maintain, can you change two things here:
* use `$SERVER` instead of "hardcoding" `spigot_server.jar`
* put a backslash and line break after each `||` and `&&` to break this long line into four such as

```
wget -q -N spigot_server.jar https://getspigot.org$URL || \
  (echo "Falling back to http, unable to contact server using https..." && \
  wget -q -N spigot_server.jar http://getspigot.org$URL) || \
  echo "Unable to download new copy of spigot server"
```